Factorial/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge">
+<meta name="viewport" content="width=device-width, initial-scale=1.0">
+<title>Factorial using JavaScript</title>
+<script src = "script.js">
+</script>
+</head>
+<body>
+<h2>Factorial of a number</h2>
+<form onsubmit= "return fun1()">
+<input type="text" id="num1" placeholder="Enter the number" class="ip"><br>
+<br>
+<input type="submit" value="submit Now"><br>
+<br>
+<div id="mess">Hello world</div>
```

Comment:
why would we need hello world here ?

Reply:
removed that unwanted code hello world Maharshi